sdk/core/azure-core/src/http/curl/curl_connection_pool_private.hpp

```diff
@@ -71,7 +71,7 @@ namespace Azure { namespace Core { namespace Http {
      *
      * @return #Azure::Core::Http::CurlNetworkConnection to use.
      */
-    static std::unique_ptr<CurlNetworkConnection> GetCurlConnection(
+    static std::unique_ptr<CurlNetworkConnection> ExtractCurlConnection(
```

Comment:
o benefit on updating private implementation names.

Reply:
Get is confusing in relationship with ownership